packages/peregrine/lib/talons/CartPage/__tests__/useCartPage.spec.js

```diff
@@ -4,15 +4,14 @@ import { createTestInstance } from '@magento/peregrine';
 import { useCartPage } from '../useCartPage';
```

Comment:
Not sure how granular can we get with testing but I feel we have to add tests around conditionals and effects. For instance, check if `isCartUpdating` is being updated on `loading` change.